proctor-common/src/main/java/com/indeed/proctor/common/Proctor.java

```diff
@@ -88,9 +87,9 @@ public static Proctor construct(
             final TestType testType = testDefinition.getTestType();
             final TestChooser<?> testChooser;
             if (TestType.RANDOM.equals(testType)) {
-                testChooser = new RandomTestChooser(expressionFactory, functionMapper, testName, testDefinition);
+                testChooser = new RandomTestChooser(RuleEvaluator.EXPRESSION_FACTORY, functionMapper, testName, testDefinition);
```

Comment:
Just curious, what's the reason to replace expressionFactory with RuleEvaluator.EXPRESSION_FACTORY? is it for improving code quality?

Reply:
TBH, there is no strong reason of this change. This is a local variable so I cant use it to build `localContext`. But I want to make it explicit that we use the same `EXPRESSION_FACTORY`.